snapshots/testsuite/testsuite.go

```diff
@@ -18,7 +18,7 @@ package testsuite
 
 import (
 	"context"
-	//nolint:golint
+	//nolint:revive // go-digest needs the blank import. See https://github.com/opencontainers/go-digest#usage.
 	_ "crypto/sha256"
```

Comment:
No need to change, but possibly the `nolint` is not needed if the comment is after the import;

```suggestion
	_ "crypto/sha256" // go-digest needs the blank import. See https://github.com/opencontainers/go-digest#usage.
```

I think the linter only complains if you use `_` imports, and there's no comment explaining why